images/win/scripts/Installers/Install-Haskell.ps1

```diff
@@ -0,0 +1,17 @@
+################################################################################
+##  File:  Install-Haskell.ps1
+##  Desc:  Install Haskell for Windows
+################################################################################
+
+
+# The latest version will be installed as a default
+$GhcVersions = @('8.6.5', '8.8.3', '8.10.1')
```

Comment:
is it possible to avoid hardcoding the versions? I worry about the further support, we will have to update versions manually on next releases.
May be there is some release .json page that can be parsed to get 3 latest versions from it?

Reply:
Hm, that is not super simple because we need to find 3 latest major-minor versions and then find latest patch for each of them but let's try.

As for the parsing source, I think this link can be used: https://downloads.haskell.org/~ghc/

Reply:
We can parse `choco search ghc --allversions` and get the latest version per branch.

Reply:
Even `choco search ghc --allversions --limit-output` to simplify parsing